src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -348,6 +453,16 @@ const NetworkModificationNodeEditor = ({ selectedNode }) => {
         [modifications, studyUuid, selectedNode.id, snackError]
     );
 
+    const isLoading = () => {
+        if (notificationList.length === 0) return false;
+
+        let res = notificationList.filter(
+            (notification) => notification?.nodeUuid === selectedNode?.id
+        );
+        if (res.length > 0) return true;
```

Comment:
You could factorize a bit this code but ok

Reply:
`return notificationIdList.filter(
            (notification) => notification === selectedNode?.id
        ).length > 0;`